pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -389,11 +389,10 @@ func (oe *operationExecutor) DetachVolume(
 }
 
 func (oe *operationExecutor) VerifyVolumesAreAttached(
-	attachedVolumes []AttachedVolume,
-	nodeName types.NodeName,
+	attachedVolumesByNode map[types.NodeName][]AttachedVolume,
 	actualStateOfWorld ActualStateOfWorldAttacherUpdater) error {
 	volumesAreAttachedFunc, err :=
-		oe.operationGenerator.GenerateVolumesAreAttachedFunc(attachedVolumes, nodeName, actualStateOfWorld)
+		oe.operationGenerator.GenerateVolumesAreAttachedFunc(attachedVolumesByNode, actualStateOfWorld)
 	if err != nil {
```

Comment:
Yeah this PR doesn't changes that behavior but when we were polling each node one by one, it was fine to do that concurrently and now when we are going to poll entire cluster in one shot - it is wasteful (and racey) to allow parallel polling here. 

We can do fix it in follow up PR or something.